docs/pubsub-usage.rst

```diff
@@ -66,7 +66,7 @@ Delete a topic:
    >>> topic = client.topic('topic_name')
    >>> topic.delete()  # API request
 
-Fetch the IAM policy for a topic:
+Update the IAM policy for a topic:
```

Comment:
May be easier to parse visually (i.e. shorter is better) if the Fetch is standalone and the Update follows immediately after